client/reader/full-post/_style.scss

```diff
@@ -112,7 +112,6 @@
 .reader__full-post-content {
 	@extend %content-font;
 	margin: 0;
-	padding-top: 16px;
```

Comment:
does this affect the current full post in a bad way? Might need to scope the change to `.is-group-reader-refresh &`